src/components/tabs/demoDynamicTabs/index.html

```diff
@@ -16,13 +16,25 @@
 
   <form ng-submit="addTab(tTitle,tContent)" flex>
     <div layout="row" layout-sm="column" layout-margin layout-align="left center">
-      <md-text-float label="Active Index" ng-model="selectedIndex" disabled></md-text-float>
-      <md-text-float label="Active Title" ng-model="tabs[selectedIndex].title"></md-text-float>
+      <md-input-container>
+        <label>Active Index</label>
+        <input type="text" ng-model="selectedIndex">
```

Comment:
I thought the same, but when i compared it with current docs:
https://material.angularjs.org/#/demo/material.components.tabs

it seems it doesn't work, so i decided to not disable the input, because i didn't want to change the behavior after refactor, but maybe i missed something.

What do you think? Should i add the disabled attr?